httplib.h

```diff
@@ -868,15 +868,16 @@ class Server {
   bool routing(Request &req, Response &res, Stream &strm);
   bool handle_file_request(const Request &req, Response &res,
                            bool head = false);
-  bool dispatch_request(Request &req, Response &res, const Handlers &handlers);
-  bool
+  static bool dispatch_request(Request &req, Response &res,
```

Comment:
I don't like to put 'static'. It's less readable.

Reply:
I understand your feeling, but having a static method means the `this` pointer is not passed as an automatic parameter. This will reduce the call overhead a little bit and ensure the calling instance won't affect the outcome or be affected by any side-effect.